src/openrct2/localisation/Language.cpp

```diff
@@ -50,7 +50,7 @@ const LanguageDescriptor LanguagesDescriptors[LANGUAGE_COUNT] =
     { "sv-SE", "Swedish",               "Svenska",               LANGUAGE_UNDEFINED, FAMILY_OPENRCT2_SPRITE,                false }, // LANGUAGE_SWEDISH
     { "tr-TR", "Turkish",               "Türkçe",                LANGUAGE_UNDEFINED, FAMILY_OPENRCT2_SPRITE,                false }, // LANGUAGE_TURKISH
     { "vi-VN", "Vietnamese",            "Vietnamese",            LANGUAGE_UNDEFINED, FAMILY(&TTFFamilySansSerif),           false }, // LANGUAGE_VIETNAMESE
-    { "uk-UA", "Ukrainian",             u8"Yкраïнська",          LANGUAGE_UNDEFINED, FAMILY_OPENRCT2_SPRITE,                false }, // LANGUAGE_UKRAINIAN
+    { "uk-UA", "Ukrainian",             u8"Украïнська",          LANGUAGE_UNDEFINED, FAMILY_OPENRCT2_SPRITE,                false }, // LANGUAGE_UKRAINIAN
```

Comment:
This is incorrect. You should never encode a Cyrillic letter as a Latin one. Shall I push a fix?

Reply:
I have created a PR of my own: https://github.com/OpenRCT2/OpenRCT2/pull/20393 , as that can be merged independently of adding the language itself.